src/components/Proposal/Create/SchemeForms/Validators.ts

```diff
@@ -0,0 +1,49 @@
+import { isHexStrict } from "web3-utils";
+import { isAddress } from "lib/util";
+
+/**
+ * Given a value returns error message in case value is less than 0 or no value provided
```

Comment:
return error string,

Reply:
I'm not sure I understand, anyway this is the way Formik handles errors in a from.

Reply:
add documentation for the return value for this function